modules/ROOT/pages/ibm/ibm-ctg-connector-reference.adoc

```diff
@@ -3,175 +3,273 @@ ifndef::env-site,env-github[]
-
-* config
+* <<config>> 
 
 ==== Throws
-
-* IBMCTG:AEI0
-* IBMCTG:CONNECTION_ERROR
-* IBMCTG:CONNECTIVITY
-* IBMCTG:ECI_ERR_INVALID_CALL_TYPE
-* IBMCTG:ECI_ERR_NO_CICS
-* IBMCTG:ECI_ERR_SECURITY_ERROR
-* IBMCTG:ECOM
-* IBMCTG:ERR_GATEWAY
-* IBMCTG:IBMCCExecution
-* IBMCTG:REQUEST_ERROR
-* IBMCTG:RESOURCE
-* IBMCTG:RETRY_EXHAUSTED
-* IBMCTG:SERVER_ERROR
-* IBMCTG:TRANSACTION
-* IBMCTG:UNKNOWN
+* IBMCTG:ERR_GATEWAY 
+* IBMCTG:REQUEST_ERROR 
+* IBMCTG:RETRY_EXHAUSTED 
+* IBMCTG:TRANSACTION 
+* IBMCTG:ECOM 
+* IBMCTG:AEI0 
+* IBMCTG:SERVER_ERROR 
+* IBMCTG:IBMCCExecution 
+* IBMCTG:CONNECTIVITY 
+* IBMCTG:CONNECTION_ERROR 
+* IBMCTG:UNKNOWN 
+* IBMCTG:ECI_ERR_INVALID_CALL_TYPE 
+* IBMCTG:ECI_ERR_SECURITY_ERROR 
+* IBMCTG:RESOURCE 
+* IBMCTG:ECI_ERR_NO_CICS 
 
 
 [[executeUsingCommarea]]
-== Execute Using COMMAREA
+=== Execute Using COMMAREA
 
 `<ibmctg:execute-using-commarea>`
 
-=== Parameters
 
+==== Parameters
 [%header%autowidth.spread]
 |===
 | Name | Type | Description | Default Value | Required
-| Configuration | String | The name of the configuration to use. | |x
-| Content a| Binary |  |  `#[payload]` |
-| Commarea Request Type a| COMMAREA Request |  |  |x
+| Configuration | String | The name of the configuration to use. | | x
+| Content a| Binary |  |  `#[payload]` | 
+| Commarea Request Type a| <<CommareaRequest>> |  |  | x
 | Transactional Action a| Enumeration, one of:
 
 ** ALWAYS_JOIN
 ** JOIN_IF_POSSIBLE
-** NOT_SUPPORTED |  The type of joining action that operations can take regarding transactions. |  JOIN_IF_POSSIBLE |
-| Streaming Strategy a| * Repeatable in memory stream
-* Repeatable file store stream
-* Non-repeatable stream |  Configure to use repeatable streams and their behavior. |  |
-| Target Variable a| String |  The name of a variable in which to store the output of the operation.  |  |
-| Target Value a| String |  An expression to evaluate against the operation's output, and the outcome of that expression is stored in the target variable. |  `#[payload]` |
-| Reconnection Strategy a| * Reconnect
-* Reconnect forever |  A retry strategy in case of connectivity errors. |  |
+** NOT_SUPPORTED |  +++The type of joining action that operations can take regarding transactions.+++ |  +++JOIN_IF_POSSIBLE+++ | 
+| Streaming Strategy a| * <<repeatable-in-memory-stream>>
+* <<repeatable-file-store-stream>>
+* non-repeatable-stream |  +++Configure to use repeatable streams.+++ |  | 
+| Target Variable a| String |  +++The name of a variable to store the operation's output.+++ |  | 
+| Target Value a| String |  +++An expression to evaluate against the operation's output and store the expression outcome in the target variable+++ |  `#[payload]` | 
+| Reconnection Strategy a| * <<reconnect>>
+* <<reconnect-forever>> |  +++A retry strategy in case of connectivity errors.+++ |  | 
+|===
+
+==== Output
+[%autowidth.spread]
+|===
+|Type |Binary
+|===
+
+=== For Configurations
+* <<config>> 
+
+==== Throws
+* IBMCTG:ERR_GATEWAY 
```

Comment:
Same comment about alphabetical order as before

Reply:
No longer alphabetizing Throws elements -- not sustainable to do so